handbook/fasse.md

```diff
@@ -192,6 +192,20 @@ data/
 *.rst
 ```
 
+## Step 8. Scratch Space
+
+Scratch space provides a dedicated area for temporary storage, improves performance, streamlines workflows, enhances data protection, enables resource isolation, and facilitates efficient clean-up. It is particularly valuable in high-performance computing (HPC), scientific research, data analysis, and other computationally intensive tasks.
+
+We have created a scratch space within our FASSE space: 
+
+```
+Directory: /n/holyscratch01/LABS/dominici_nsaph/Lab 
+```
+
+Scratch space provides a dedicated area for temporary storage and facilitates efficient data processing. It allows you to perform tasks such as data preprocessing, computationally intensive computations, and analysis, while storing intermediate files and temporary results. The high-performance storage and resource isolation capabilities of a scratch space enhance performance, streamline workflows, and protect valuable data. It is particularly useful in high-performance computing environments, scientific research, and data-intensive tasks where speed, efficiency, and data management are critical.
+
```

Comment:
We would like to include instructions on how to use the scratch folder. 

@keziatesiman If you cd into this path you will see that it contains /projects and /data_processing folders. 


Under projects there can be user-subfolders or project-subfolders.

Ideally a symlink the path folder should be incorporated within the project repo that is using the scratch space and not full paths.

Can you put together instructions with your own words?

Reply:
i think both instructions on how to use (where to save etc) and also when to use would be useful, thanks!

Reply:
I will incorporate some edits.